modify-keybinds.md

```diff
@@ -23,6 +23,6 @@ Jupyterlab commands don't know about the concept of vim modes so they will be ac
 
 ### Vim remappings
 
-Vim style remappings (`inoremap`, `imap`, `nmap`...) can be modified in the advanced settings editor under the `Notebook Vim` settings. Alternatively, you can use the sibling extension [jupyterlab-vimrc](https://github.com/ianhi/jupyterlab-vimrc#jupyterlab-vimrc) to do this.
+Vim style remappings (`inoremap`, `imap`, `nmap`...) can be modified in the settings editor under the `Notebook Vim` settings. Alternatively, for JupyterLab <4 you can use the sibling extension [jupyterlab-vimrc](https://github.com/ianhi/jupyterlab-vimrc#jupyterlab-vimrc) to do this.
```

Comment:
```suggestion
Vim style remappings (`inoremap`, `imap`, `nmap`...) can be modified in the Settings Editor under the `Notebook Vim` settings. Alternatively, for JupyterLab <4 you can use the sibling extension [jupyterlab-vimrc](https://github.com/ianhi/jupyterlab-vimrc#jupyterlab-vimrc) to do this.
```

@ianhi does jupyterlab-vimrc add something on top, or should we drop the mention of it?

Reply:
It added the ability to use `set unnamedplus=True` which is tracked in #79, as well it allowed the `+` register to yank to the system clipboard which is now available with the replit codemirror!

So for jlab3 it has a real value add w.r.t copy behavior, but does not for jlab4 - and I don't plan on updating it for jlab4 as it's major functionality is now handled here.

I think a more accurate statement is now that for Jlab <4 it provides yanking to the system clipboard and the ability to set unnamedplus.